inference/README.md

```diff
@@ -117,3 +122,16 @@ cd text-client
 pip install -r requirements.txt
 python __main__.py
 ```
+
+## Distributed Testing
+
+We run distributed load tests using the
+[`locust`](https://github.com/locustio/locust) Python package.
+
+```bash
+pip install locust
+cd tests/locust
+locust
+```
```

Comment:
Since 436dd9b478c238fc5eb6d3ca641c2525fb0b056d was merged the endpoints by default require authentication to call. To test without authenticating currently would need to set `use_auth=False` in the inference server settings, so we may want to document that here.

Reply:
small update, there's now a debug auth setting that lets you log in with arbitrary user names, see the text client for an example

Reply:
Thanks, I've copied over the new implementation into the locust test. I think for the time being that's okay and in the future we can think about making utility functions from the text client installable or something?